src/wp-admin/options-general.php

```diff
@@ -124,9 +124,12 @@
 	$app_icon_alt_value     = '';
 	$browser_icon_alt_value = '';
 
+	// Store icon URL as variables for use in multiple places.
```

Comment:
Nit: this is not really a useful comment. The code is self-explanatory as-is.

Reply:
Ok!